### PR TITLE
Support image_tag_pattern with autoselection of latest

### PR DESCRIPTION
**Pull request by heubeck**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 FROM google/cloud-sdk:298.0.0-alpine
 
 COPY entrypoint.sh /entrypoint.sh
-RUN chmod a+x /entrypoint.sh
+RUN apk add jq && \
```
Comment (kunzese) [inline Dockerfile]:
```suggestion
RUN apk add --no-cache jq && \
```

Comment (heubeck) [reply Dockerfile]: zu langsam ;)

File: README.MD [hunk 2]
```diff
@@ -14,15 +14,24 @@ A Github Action that deploys a service to Google Cloud Run (GCP managed Knative-
...
 | `memory` | Memory limit per instance | `256Mi` | false | [gcloud run deploy](https://cloud.google.com/sdk/gcloud/reference/run/deploy#--memory) |
 | `max_instances` | Max nummber of instances to be scaled | `10` | false | [gcloud run deploy](https://cloud.google.com/sdk/gcloud/reference/run/deploy#--max-instances) |
 | `request_timeout` | Timeout for a single request to be processed | `10s` | false | [gcloud run deploy](https://cloud.google.com/sdk/gcloud/reference/run/deploy#--timeout) |
 | `allow_unauthenticated` | Wether the service should not be protected by GCP authorization | `true` | false | [gcloud run deploy](https://cloud.google.com/sdk/gcloud/reference/run/deploy) |
 | `service_account` | Service Account to be used by the revision to be deployed | GCP docs: "If not provided, the revision will use the project's default service account." | false | [gcloud_run_deploy](https://cloud.google.com/sdk/gcloud/reference/run/deploy#--service-account) |
+| `no_traffic` | Set to true for just deploying a new revision without shifting traffic | `false` | false | [gcloud run deploy](https://cloud.google.com/sdk/gcloud/reference/run/deploy#--no-traffic) |
```
Comment (kunzese) [inline README.MD:25]:
```suggestion
| `no_traffic` | Set to true to just deploy a new revision without shifting traffic | `false` | false | [gcloud run deploy](https://cloud.google.com/sdk/gcloud/reference/run/deploy#--no-traffic) |
```

File: action.yml [hunk 3]
```diff
@@ -52,6 +56,10 @@ inputs:
     description: 'Service Account to be used by the revision to be deployed'
     required: false
     default: 'default'
+  no_traffic:
+    description: 'Set to true for just deploying a new revision without shifting traffic'
```
Comment (kunzese) [inline action.yml:60]:
```suggestion
    description: 'Set to true to just deploy a new revision without shifting traffic'
```

**Comment by kunzese** [comment]
hmm the Markdown linter failed with after the merge.

`standard_init_linux.go:211: exec user process caused "no such file or directory"`

**Pull request merged**